bloom-housing/bloom
# fix: listing duplicate fe

This PR addresses [#773](https://github.com/metrotranscom/doorway/issues/773)


## Description

This PR adds the FE component to the listing duplication feature based on the designs in the attached ticket. Additionally, this PR adds the created date and application due date to the listings table on the partners side.

Update: Additionally, this PR updates the BE logic to support a duplicate listing permissioning array which mirrors the pattern of the listing approval array. Lastly, there is a new feature flag to determine how limited the options on a closed listing should be (See AC updates at the bottom of the linked ticket.)

## How Can This Be Tested/Reviewed?

This can be reviewed by running this locally, clicking into any listing, clicking copy, and ensure that the form requires an original name and the includeUnits checkbox is functional. 

Additionally, try turning the feature flag LIMIT_CLOSED_LISTING_ACTIONS on/off or updating the duplicateListingPermissions array on the jurisdictions table.
 
## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
